File: conversions/django_models/collective.py
from .abstract_element_model import AbstractElementModel
from django.db import models

class Collective(AbstractElementModel):

    # Formation
    composition = models.TextField(blank=True, null=True)
    count = models.PositiveIntegerField(blank=True, null=True)
    formation_date = models.PositiveIntegerField(blank=True, null=True)
    operator = models.ForeignKey("Institution", on_delete=models.SET_NULL, blank=True, null=True, related_name="collective_operator")
    equipment = models.ManyToManyField("Construct", blank=True, related_name="collective_equipment")

    # Agency
    activity = models.TextField(blank=True, null=True)
    temperance = models.TextField(blank=True, null=True)
    skills = models.ManyToManyField("Ability", blank=True, related_name="collective_skills")
    rituals = models.ManyToManyField("Construct", blank=True, related_name="collective_rituals")

    # World
    species = models.ManyToManyField("Species", blank=True, related_name="collective_species")
    characters = models.ManyToManyField("Character", blank=True, related_name="collective_characters")
    creatures = models.ManyToManyField("Creature", blank=True, related_name="collective_creatures")
    phenomena = models.ManyToManyField("Phenomenon", blank=True, related_name="collective_phenomena")

    def __str__(self):
        return self.name

File: conversions/csharp_unity_json_properties/Collective.cs
using System;
using System.Collections.Generic;
using Newtonsoft.Json;
using Utils;

[System.Serializable]
public class Collective : Element
{
    [JsonProperty("composition"), TextAttribute("")]
    public string composition;
    [JsonProperty("count"), Integer(0)]
    public int count;
    [JsonProperty("formation_date"), Integer(0)]
    public int formationDate;
    [JsonProperty("operator"), ReferenceAttribute(typeof(Institution))]
    public string operator;
    [JsonProperty("equipment"), ReferenceAttribute(typeof(Construct), true)]
    public string equipment;
    [JsonProperty("activity"), TextAttribute("")]
    public string activity;
    [JsonProperty("temperance"), TextAttribute("")]
    public string temperance;
    [JsonProperty("skills"), ReferenceAttribute(typeof(Ability), true)]
    public string skills;
    [JsonProperty("rituals"), ReferenceAttribute(typeof(Construct), true)]
    public string rituals;
    [JsonProperty("species"), ReferenceAttribute(typeof(Species), true)]
    public string species;
    [JsonProperty("characters"), ReferenceAttribute(typeof(Character), true)]
    public string characters;
    [JsonProperty("creatures"), ReferenceAttribute(typeof(Creature), true)]
    public string creatures;
    [JsonProperty("phenomena"), ReferenceAttribute(typeof(Phenomenon), true)]
    public string phenomena;
}
